Automattic/jetpack
# Instant Search: fix warnings appearing in browser console

#### Changes proposed in this Pull Request:

I'm opening this as a follow-up to #20369, to address some minor warnings that came up as I was testing this PR.

#### Jetpack product discussion

* No

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Apply this change to your site with a Jetpack Search subscription.
* Navigate to the new customization page at `/wp-admin/admin.php?page=jetpack-search-customize`.
* Check the browser console for warnings.
